proc-macros/tests/ui/correct/basic.rs

```diff
@@ -13,6 +13,15 @@ pub trait Rpc {
 	#[method(name = "foo")]
 	async fn async_method(&self, param_a: u8, param_b: String) -> JsonRpcResult<u16>;
 
+	#[method(name = "optional_params")]
+	async fn optional_params(&self, a: Option<u8>, b: String) -> JsonRpcResult<bool>;
```

Comment:
Not sure why anybody would, but it may also be useful to test that `std::option::Option` works, and maybe test that something stupid like `foo::Option::Bar` doesn't work?